features/support/github.js

```diff
@@ -36,17 +36,25 @@ function randomString(stringLength) {
  * @param  {String}          repoOwner  Owner of the repository
  * @param  {String}          repoName   Name of the repository
  * @param  {String}          branch     Name of the branch to delete
+ * @param  {String}          tag        Name of the tag to delete
  * @return {Promise}
  */
-function cleanUpRepository(branch, repoOwner, repoName) {
+async function cleanUpRepository(branch, tag, repoOwner, repoName) {
```

Comment:
This function seems to not return any `Promise`. Is it ok?

Reply:
Same as above. I fixed it.